README.md

```diff
@@ -40,12 +55,16 @@ This doesn't necessarily deactive the environment. To do this you must the follo
 
 `deactivate`
 
-### Template Info
-#### Project structure 
+## Data 
+We have python codes to prepare the data. The code contains two functions. The first function splits the data into training and testing sets. The second function normalizes the data. The data was very clean from Kaggle.
```

Comment:
I would say we have python "modules" not "codes"

Reply:
Changed to "modules"